pkg/operation/shoot/types.go

```diff
@@ -90,9 +90,10 @@ type Components struct {
 
 // ControlPlane contains references to K8S control plane components.
 type ControlPlane struct {
-	KubeAPIServerService component.DeployWaiter
-	KubeAPIServerSNI     component.DeployWaiter
-	KubeScheduler        kubescheduler.KubeScheduler
+	KubeAPIServerService      component.DeployWaiter
+	KubeAPIServerServicePhase component.Phase
```

Comment:
```suggestion
	KubeAPIServerSNIPhase component.Phase
```
since it's only SNI related